API/NoDocs/Microsoft.Extensions.Http.AutoClient/GetAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Microsoft.Extensions.Http.AutoClient'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Http.AutoClient;
+
+[AttributeUsage(AttributeTargets.Method)]
+public sealed class GetAttribute : Attribute
```

Comment:
Yes, these are for HTTP methods. Unfortunately I don't think we can align, because the existing one in aspnet has `AllowMultiple = true` which in the use case of the autoClient wouldn't apply.